suite-native/toast-notifications/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@suite-native/toast-notifications",
```

Comment:
Please drop `notifications` from all names everywhere. Just `toasts` is enough.

Reply:
9c3195cce